arithmetic-formatter/arithmetic_arranger.py

```diff
@@ -0,0 +1,33 @@
+def arithmetic_arranger(problems, displayMode=False):
+    first_operand_line = second_operand_line = dashes_line = result_line = ""
+    result_new_line = "\n"
+    if len(problems) > 5:
+        return "Error: Too many problems."
+    for index in range(len(problems)):
+        problem = problems[index]
+        [operand1, operator, operand2] = problem.split()
+
+        # Error Handling
+        if len(operand1) > 4 or len(operand2) > 4:
+            return "Error: Numbers cannot be more than four digits."
+        if operator != "+" and operator != "-":
+            return "Error: Operator must be '+' or '-'."
+        if not operand1.isnumeric() or not operand2.isnumeric():
+            return "Error: Numbers must only contain digits."
+
+        operand_max_length = max(len(operand1), len(operand2))
+        before_line_spaces = operand_max_length + 2
+
+        if index == 0:
+            spaces_before_result_dashes = ""
+        else:
+            spaces_before_result_dashes = " " * 4
+            result_new_line = ""
```

Comment:
Currently variable name is showing that it will always add the new line which is not what we have in code ..... `result_line_prefix` seems like a better name

Reply:
You have changed the wrong variable name ... kindly change this one and keep the other one same as before